.clitest/znap.source.find.md

```diff
@@ -0,0 +1,63 @@
+Initialize
+```zsh
+% source .clitest/__init__.zsh
+% .test.setup
+%
+```
+
+Source finds correct plugin file
+```zsh
+% dir=$T_TEMPDIR
+% .test.makeplugin foo.plugin.zsh
+% reply=(); .znap.source.find $dir/foo  #=> --exit 0
```

Comment:
I would prefer if you could use line breaks instead of `;` (except in constructs like `if ...; then`, `for ...; do`, etc.)